plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -505,6 +505,38 @@ SecRule REQUEST_FILENAME "@contains /settings/personal/authtokens/" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
 
+# When trying to update availability in Personal --> Availability
+SecRule REQUEST_FILENAME "@contains /remote.php/dav/calendars/admin/inbox" \
+    "id:9508602,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_request_content_type=%{tx.allowed_request_content_type} |text/plain|'"
+
+
+# While going through the first run setup wizard
+SecRule REQUEST_FILENAME "@contains /apps/firstrunwizard/wizard" \
```

Comment:
You could probably use `@beginsWith` instead of `@contains` for all the rules here that use `@contains`.

Reply:
The problem with `@beginsWith` is that some installations might install Nextcloud in a subfolder in relation to the webroot, and the above will look something like `/nextcloud/apps/firstrunwizard/wizard`, in which case this fix will not work. So, it's better to use `@contains` or `@endsWith` in this situation.